docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,53 @@
+
+[discrete]
+== Download and modify the daemonset manifest
+The daemonset integrates Elastic Endpoint into your Kubernetes cluster. The Elastic Agent is enrolled to a running Fleet Server using the `FLEET_URL` parameter, and connected to a specific Elastic Agent policy using the `FLEET_ENROLLMENT_TOKEN`.
+
+1. Download the daemonset manifest http://github.com/elastic/placeholder[here] for GKE and http://github.com/elastic/placeholder[here] for EKS.
+2. In the manifest, fill in the `FLEET_URL` field with your Fleet URL. If you’re not sure how to find it, refer to https://www.elastic.co/guide/en/fleet/current/fleet-settings.html[Fleet UI settings].
+3. In the manifest, fill in the `FLEET_ENROLLMENT_TOKEN` field with your Fleet enrollment token. If you don’t have one or you’re not sure how to find it, refer to https://www.elastic.co/guide/en/fleet/master/fleet-enrollment-tokens.html[Fleet enrollment tokens].
+
+[discrete]
+== Apply the modified manifest to your cluster or clusters
+First, to ensure you install Endpoint on the desired k8s cluster(s), use `kubectl config use-context <name-of-context>` to set the default context. You can repeat the following process for multiple contexts.
+
+NOTE: To check which contexts exist, use `kubectl config get-contexts` to list them from your local kubectl config file. An asterisk, if present, indicates the current default context.
+
+To apply the manifest to a cluster, run for example: `kubectl apply -f eks-elastic-security.yaml` for EKS, or `kubectl apply -f gke-elastic-security.yaml` for GKE.
+
+To check the daemonset’s status, run for example: `kubectl get pods -A`
+
+Once the daemonset is running, Elastic Endpoint will start sending Linux session data from Kubernetes to Kibana. You can then view that data using the Kubernetes dashboard.
+
+[discrete]
+== The Kubernetes dashboard
+To find the dashboard, go to **Security > Dashboards > Kubernetes**.
+
+image::kubernetes-dashboard.png[the Kubernetes dashboard, with numbered labels 1 through 3 for major sections]
+Sections numbered above are described below:
+
+  1. The charts at the top of the dashboard provide an overview of your monitored Kubernetes infrastructure. You can hide them by clicking Hide charts.
+  2. The Tree navigation menu allows you to navigate through your deployments and select the scope of the Sessions table to the right. You can select any item in the menu to show its sessions. In Logical view, the menu is organized by Cluster, Namespace, Pod, and Container image. In Infrastructure view, it is organized by Cluster, Node, Pod, and Container image.
+  3. The sessions table displays sessions collected from the selected element of your Kubernetes infrastructure. You can view it in fullscreen by selecting the button in its upper right corner, and sort it by any of its fields.
+
```

Comment:
Should we add some text guiding the user to the metadata tab in the session view's details panel? We have added a fair amount of new Kubernetes contextual data that will give the user even more context than those shown in the tree.

Reply:
@benironside the ECS fields are listed here: https://docs.google.com/spreadsheets/d/1JEmcKvWwHX1YEaqPDEm0So8l_1WdVxONuPTe-v0gxzw/edit#gid=1933739703 and displayed in the repurposed "Metadata" tab within the details panel

Reply:
Added a section with this info, thanks.